src/test/kotlin/calculator/CalculatorTest.kt

```diff
@@ -0,0 +1,58 @@
+package calculator
+
+import calculator.domain.Calculator
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.MethodSource
+import java.util.stream.Stream
+
+class CalculatorTest {
+
+    private val calculator = Calculator()
+
+    @Test
+    fun `숫자들을 합할때`() {
+        val numbers = listOf("1", "2", "3", "4")
+
+        assertThat(calculator.calculate(numbers)).isEqualTo(10)
+    }
+
+    @Test
+    fun `숫자들중 음수가 포함되어 있을때`() {
+        val numbers = listOf("1", "2", "3", "-4")
+
+        assertThrows<RuntimeException>("음수는 계산식에 포함될 수 없습니다.") { calculator.calculate(numbers) }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        assertThrows<RuntimeException> { calculator.calculate(numbers) }
```